FogCreek/Glitch-Community
# Gossamer maize: fix 'create team'

## Links
* Remix link: https://gossamer-maize.glitch.me/
* Issue-tracker link: https://app.clubhouse.io/glitch/story/6038/getting-a-team-already-exists-error-when-trying-to-create-a-team-with-any-custom-name

## Changes:
* fix checks for existing teams/users with the given name
* remove isVerified flag from create team payload

## How To Test:
* log into https://gossamer-maize.glitch.me/
* trying to create a team called "jungle-order" should show an error
* trying to create a team with an unused name should work
* trying to create a team with the default name should work

## Things left to do before deploying:
- [ ] un-revert the reverted PR
